debian/copyright

```diff
@@ -3,7 +3,8 @@ Upstream-Name: http-relay
 Source: <https://github.com/ctu-vras/http_relay>
 
 Files: *
-Copyright: Czech Technical University in Prague
+Copyright: 2023 Martin Pecka <peci1@seznam.cz>,
```

Comment:
Are the years actually needed? I never understood their value (maybe in the very old ages without VCS, it could tell who was working on the code when, but now we have full history). It seems it is just another thing that pollutes the build files and complicates maintenance (updating the years ranges etc). I haven't found any requirement for the years in the Debian packaging policy. It says it is freeform text.

Also, I work for Czech Technical University, so I don't need to be explicitly attributed as an author. I just used my personal email address for the package as that is more persistent than a work email (and I intend to maintain the package even if I leave).

Reply:
> Are the years actually needed? I never understood their value (maybe in the very old ages without VCS, it could tell who was working on the code when, but now we have full history). It seems it is just another thing that pollutes the build files and complicates maintenance (updating the years ranges etc). I haven't found any requirement for the years in the Debian packaging policy. It says it is freeform text.

I'm afraid that as far as I can tell, the copyright years are mandatory, lintian will emit a [warning](https://lintian.debian.org/tags/copyright-without-copyright-notice) in the current state:
```
Now running lintian http-relay_2.1.1-4_amd64.changes ...
W: http-relay: copyright-without-copyright-notice
W: python3-http-relay: copyright-without-copyright-notice
Finished running lintian.

```

Reply:
Okay, then it's inevitable...